### ISSUE TITLE
What is the best way to use the interface with as little dependencies as possible?

### ISSUE THREAD

**Issue by mirkea** [mirego/accent#239]
Hi. 
I've looked at a bunch of tools to handle translations and Accent seems the best by far. However our tech lead feels like it comes with "too much baggage", and as it would need to be always running it would incur additional costs. 

In an ideal world our setup would be something like
- someone who needs to access the translation tool goes to a predefined URL
- they authenticate with google
- the Accent app would need to start (so it wouldn't be always on)
- the files to be translated would be fetched (we could make some API calls on start to fetch them)
- the files would be translated in Accent, then synced
- the user would log out and Accent would stop running

Now, this would probably mean that we'd lose stuff like edit history, but we're just looking for a simple and intuitive UI to load up and translate the files in.

So basically, the question is:  what is the best way to use the interface with as little dependencies as possible?

**Comment by louim** [comment]
Hey @mirkea,

I'm not a developper, only a happy user of Accent giving you my honest opinion. The engineering cost of running Accent _on demand_ would far outweigh the cost of running it continually. This is the cost of running Accent on Heroku. It fits my definition of cheap. You could probably get even cheaper prices if you run it on a single VPS (like Digital ocean or Hetzner), but you would have to do the setup yourself.

<img width="1231" alt="Screen Shot 2021-03-17 at 10 11 57 AM" src="https://user-images.githubusercontent.com/923718/111481836-b705d400-8709-11eb-8e0e-1b192e8f0b88.png">

**Comment by simonprev** [comment]
Thanks for the reply @louim , you’re right, the goal of Accent is to be simple and cost-efficient to run. I could write a more detailed guide on how to run Accent on the cheapest platform here https://www.accent.reviews/guides/deploy.html
